pywhat/Data/regex.json

```diff
@@ -727,6 +727,19 @@
          "Credentials",
          "Password",
          "Username"
-       ]
-    }
-]
+      ]
+   },
+   {
+      "Name": "PGP Public Key",
+      "Regex": "/^(-----BEGIN PGP PUBLIC KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PUBLIC KEY BLOCK-----)$|^(-----BEGIN PGP PRIVATE KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PRIVATE KEY BLOCK-----)$/",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 0,
+      "URL": "https://8gwifi.org/pgpencdec.jsp",

```

Comment:
The way the url works is that the matched regex will go to https://8gwifi.org/pgpencdec.jsp, which means if the regex is "-----BEGIN PGP PUBLIC KEY BLOCK-----' it'll go to: 

```
https://8gwifi.org/pgpencdec.jsp-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Reply:
Thanks, I removed the URL parameter as I had misunderstood how it worked and I don't know about a tester that can accept parameters on the URL itself